pkgs/development/tools/build-managers/bloop/default.nix

```
@@ -1,72 +1,77 @@
-{ stdenv, lib, fetchurl, coursier, python, makeWrapper }:
-
-let
-  baseName = "bloop";
-  version = "1.3.4";
-  nailgunCommit = "d7ed5db"; # Fetched from https://github.com/scalacenter/bloop/releases/download/v${version}/install.py
-
-  client = stdenv.mkDerivation {
-    name = "${baseName}-client-${version}";
+{ stdenv, fetchurl, coursier,
+ autoPatchelfHook, 
+ lib, zlib }:
```

```suggestion
{ stdenv
, fetchurl
, coursier
, autoPatchelfHook
, lib
, zlib 
}:
```

I personally think this looks/works best

---

Done